apps/kyb-app/entrypoint.sh

```
@@ -0,0 +1,46 @@
+#!/usr/bin/env sh
+
+if [[ -z "$VITE_DOMAIN" ]]
+then
+    VITE_DOMAIN="http://localhost:3000"
+fi
+
+if [[ -z "$VITE_API_KEY" ]]
```

Shellcheck

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)